docs/troubleshooting/index.md

```diff
@@ -1 +1,19 @@
 # Troubleshooting
+
+Some common problems and resolutions
+
+## Ports 9001-9004 aren't open after starting m3db.
+
+These ports will not open until a namespace and placement have been created and the nodes have bootstrapped.
+
+## Bootstrapping is slow
+
+Double check your configuration against the [bootstrapping guide](../operational_guide/bootstrapping.md). The nodes will log what bootstrapper they are using and what time range they are using it for.
+
+If you don't seem to be using the filesystem bootstrapper, ensure that snapshotting is enabled for your namespace.
```

Comment:
I'm a little confused by this sentence. What do you mean by "if you don't seem to be using the filesystem bootstrapper"? Snapshotting actually makes the commitlog boostrapper faster, but doesn't have any impact on the filesystem bootstrapper (which is reading immutable files, not the snapshots)